### PR TITLE
fix(select): missing aria-expanded on the template

### PR DESCRIPTION
**Pull request by m4thieulavoie**
# Pull Request

## 📖 Description

This PR adds the `aria-expanded` to the template of the Select. Since the element has the `role="combobox"`, this is required. Axe was also complaining about it

### General

- [x] I have included a change request file using `$ yarn change`
- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [x] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Comment by m4thieulavoie** [comment]
CI fails, but doesn't seem to be exactly related to my change. Do I need to fix that?

**Comment by Garbee** [comment]
See #4492 on Code Climate stuff. If that's all that is complaining I wouldn't go after it unless requested by a reviewer.

From the looks of it these alerts are due to duplicate code, which could be cumbersome to abstract. I wouldn't stress about these myself.

**Comment by EisenbergEffect** [comment]
Yeah, don't worry about the code climate stuff too much. We'll do a "human" review which will be the meaningful feedback.

File: change/@microsoft-fast-foundation-d034d172-d3c1-4cea-aea5-142253858f29.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix(select): missing aria-expanded on the template",
```
Comment (chrisdholt) [inline change/@microsoft-fast-foundation-d034d172-d3c1-4cea-aea5-142253858f29.json:3]: Thanks for leaning in with the new changes and adding the change file!

```suggestion
  "comment": "add missing aria-expanded to the select template",
  ```

Comment (m4thieulavoie) [reply change/@microsoft-fast-foundation-d034d172-d3c1-4cea-aea5-142253858f29.json]: Done!

**Review by radium-v — APPROVED** [review]

**Review by EisenbergEffect — APPROVED** [review]

**Review by chrisdholt — APPROVED** [review]

**Comment by awentzel** [comment]
I've deactivated the display of these informational notices on GitHub.  I'm putting in the backlog to verify these are removed. I also have had a ticket outstanding for a few weeks since this RFC was written and awaiting a response to get more information on usage including additional items I've been curious about.

**Pull request merged**